mlflow/pytorch/__init__.py

```diff
@@ -303,9 +303,9 @@ def load_model(model_uri, **kwargs):
                       - ``s3://my_bucket/path/to/model``
                       - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
 
-                      For more information about supported URI schemes, see the
-                      `Artifacts Documentation <https://www.mlflow.org/docs/latest/tracking.html#
-                      supported-artifact-stores>`_.
+                      For more information about supported URI schemes, see
+                      `Artifact Stores <https://www.mlflow.org/docs/latest/tracking.html#
```

Comment:
Based on above suggestion about renaming the `Artifact Store Locations` to `Referencing Artifacts`, this link should be renamed.